packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -402,6 +444,88 @@ export class NatGateway extends Resource implements IRouteTarget {
   }
 }
 
+/**
+ * Creates a peering connection between two VPCs
+ * @resource AWS::EC2::VPCPeeringConnection
+ */
+export class VPCPeeringConnection extends Resource implements IRouteTarget {
+
+  /**
+   * The type of router used in the route.
+   */
+  readonly routerType: RouterType;
+
+  /**
+   * The ID of the route target.
+   */
+  readonly routerTargetId: string;
+
+  /**
+   * The VPC peering connection CFN resource.
+   */
+  public readonly resource: CfnVPCPeeringConnection;
+
+  constructor(scope: Construct, id: string, props: VPCPeeringConnectionProps) {
+    super(scope, id);
+
+    const region = props.acceptorRegion ? props.acceptorRegion : Stack.of(this).region;
```

Comment:
this expression can be written as following:

```suggestion
    const region = props.acceptorRegion ?? Stack.of(this).region;
```

Reply:
Updated in files where relevant!